src/index.html

```diff
@@ -15,8 +15,98 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="image">
+        <a href="index.html">
+          <img
+            src="images/logo.png"
+            alt="logo_moyo"
+          />
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="list">
+          <li class="list_item">
+            <a
+              href="index.html"
+              class="link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="list_item">
+            <a
+              href="index.html"
+              class="link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="list_item">
+            <a
+              href="index.html"
+              class="link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="list_item">
+            <a
+              href="index.html"
+              class="link"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="list_item">
+            <a
+              href="index.html"
+              class="link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="list_item">
+            <a
+              href="index.html"
+              class="link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="list_item">
+            <a
+              href="index.html"
+              class="link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="list_item">
+            <a
+              href="index.html"
+              class="link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
```

Comment:
The class names should represent the meaning of the content, not the styles or tag names. For example, 'nav__list' for the 'ul' element and 'nav__item' for the 'li' elements.